examples/documents/example.proto

```diff
@@ -185,3 +185,94 @@ message Integers {
   sfixed32 valueI = 9;
   sfixed64 valueJ = 10;
 }
+
+
+/*
+The following message types are for converter tests only
+*/
```

Comment:
Are the other messages completely unusable for the converter tests? I'd love to have not too many messages and reuse the same ones (also to make sure we test with the same data across everything)